extension/Descriptors/SubmissionStatusDescriptor.xml

```diff
@@ -78,4 +78,16 @@
 		<Description>Student chose to "unsubmit" the assignment.</Description>
 		<Namespace>uri://ed-fi.org/edfilms/SubmissionStatusDescriptor/Google</Namespace>
 	</SubmissionStatusDescriptor>
+    <SubmissionStatusDescriptor>
+		<CodeValue>LATE</CodeValue>
```

Comment:
Is it necessary to put it in uppercase for the CodeValue column? I think it is easier to read, join and query when the values match in the different columns, so when possible I like to have them written in the same way,

Reply:
The original code values were copied directly from the source systems, preserving the case in from the source system documentation. In this situation we are making up the values and I have no preference. We do assume that SQL Server has been setup with the default case insensitive collation.

Reply:
Yeah, I'm following the upper case nature of the statuses that come from the API. If we want to do something different, we should probably have a bigger discussion around normalization of incoming values.